src/types.ts

```diff
@@ -509,3 +492,66 @@ export type PermissionsListEntry = {
   resource_name: string;
   uri: string;
 };
+
+export type CollectionsInfo = {
```

Comment:
Can we rename this to `SignoffCollectionsInfo` ?

Reply:
[2cbd744](https://github.com/Kinto/kinto-admin/pull/2360/commits/2cbd7443fca129ef42e0e611c75661551d5ebd7a)